src/site/content/en/fast/reduce-network-payloads-using-text-compression/index.md

```diff
@@ -156,3 +158,35 @@ const app = express();
+JavaScript is a bit tricky, as it depends on what your JavaScript resources are providing for your website.
+
+- If your JavaScript is providing interactivity for your website, minification and compression will ensure that your website's [First Input Delay (FID)](/fid/) and [Interaction to Next Paint (INP)](/inp/) metrics may be improved during startup. However, it's also important that you're only serving the JavaScript that a given page needs, otherwise you may risk delays in responding to user inputs during that crucial startup period.
+- If your JavaScript is providing markup for a page, then the situation changes. As always, smaller payloads will download faster. In the case of JavaScript, reducing resource load time means you're going to arrive at the parsing, compiling, and execution steps sooner. This may result in faster LCP times if the client-rendered markup contains the page's LCP element. Beyond that, you should try to avoid rendering markup on the client, as [it defeats the preload scanner](/preload-scanner/#rendering-markup-with-client-side-javascript). If you can't avoid client-rendering of markup, then you should preload that render-critical piece of JavaScript so that it can arrive to the browser as fast as it possibly can.
+
+### SVG images
+
+Raster image types are inherently compressed, be it lossy or lessless formats. However, SVG—being a vector image format and text-based—can benefit from compression and minification in the same way that other text-based resources can.
+
+The primary benefit of compression and minifying SVG images depends on whether the SVG image is the LCP element for the given page. If this is the case, you could expect to see lower LCP times.
+
+### Caveats on dynamic versus static compression
+
+Dynamic compression means that the resource is compressed on the server at the time of the request. The inherent nature of this type of compression is that this process takes time on the server to complete, which can delay the resource's [Time to First Byte](/ttfb/). This delay means that—depending on the compression algorithm and level—dynamic compression could actually _delay_ the resource download time.
+
+For static assets such as CSS, JavaScript, and images, you should try to avoid dynamic compression. Static compression is an alternative compression method where the resource is compressed ahead of time and stored on disk, which means that there's no server time involved in compressing the resource at the time of the request. It's the best of both worlds: you get reduced resource payloads, without any of the delays that come with dynamic compression.
+
+The only exception to this rule is with dynamically generated HTML. This is HTML that changes based on the user, such as HTML payloads generated for authenticated users. You can't statically compress this type of markup. However, if you're running a website that doesn't perform personalization of markup on the server, you can safely compress it statically.
```

Comment:
I think this is too strong. Dynamic-based compression is fine and barely noticeable at mid-level settings, unless your server is severely under powered.

A low-level compression settings it pointless.
At high-level compression settings it can cause a performance issue.

Current wording is aimed at the dangers of high-level compression, but makes it looks like it applies to all.

I'd reword this to emphasis the dangers of too high a dynamic compression level (and also the pointless-ness of too low a compression level) instead, and that mid-level is usually the good trade off.

And that static compression allows the best of both worlds as you say.

Could also be good to reference the recommended settings in this table: https://almanac.httparchive.org/en/2021/compression#fig-10